pytorch/buildspec.yml

```diff
@@ -1,8 +1,8 @@
 account_id: &ACCOUNT_ID <set-$ACCOUNT_ID-in-environment>
 region: &REGION <set-$REGION-in-environment>
 framework: &FRAMEWORK pytorch
-version: &VERSION 1.9.0
-short_version: &SHORT_VERSION 1.9
+version: &VERSION 1.8.2
```

Comment:
Create a new file `buildspec-1-8-2.yml` to store the v1.8.2 configuration and revert the changes in `pytorch/buildspec.yml` to reflect the latest PT version v1.9.0

Reply:
Done